src/devtools/client/debugger/src/actions/sources/newSources.ts

```diff
@@ -63,28 +63,21 @@ function checkSelectedSource(cx: Context, sourceId: string): UIThunkAction {
     const state = getState();
     const pendingLocation = getPendingSelectedLocation(state);
 
-    if (!pendingLocation || !pendingLocation.url) {
+    if (!pendingLocation) {
       return;
     }
 
     const source = getSource(state, sourceId);
 
-    if (!source || !source.url) {
+    if (!source) {
       return;
     }
 
-    const pendingUrl = pendingLocation.url;
-    const rawPendingUrl = getRawSourceURL(pendingUrl);
```

Comment:
AFAICT this no longer does anything because our URL's don't have `:formatted` appended to the end? (AFAIK)